### PR TITLE
Support for 2019 Subaru Ascent

### PR DESCRIPTION
**Pull request by martinl**
**Checklist**

- [x] added to README
- [x] test route added to [test_car_models](../blob/master/selfdrive/test/test_car_models.py)
- [x] route with openpilot: 3c8f0c502e119c1c|2020-06-30--12-58-02
- [ ] route with stock system:

Tested and working on 2019 Subaru Ascent Limited (Adminiuga).

PS. Tuning was done by ndtran on 2020 Ascent. 2020 Ascent fingerprints as Impreza, so we can only upstream 2019 until we implement FPv2 for Subaru

### CODE REVIEW

File: selfdrive/car/subaru/interface.py [hunk 1]
```diff
@@ -28,6 +28,17 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
     ret.steerRateCost = 0.7
     ret.steerLimitTimer = 0.4
 
+    if candidate == CAR.ASCENT:
+      ret.safetyModel = car.CarParams.SafetyModel.subaru
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/interface.py:32]: This isn't necessary as it's set as the default above

Comment (martinl) [reply selfdrive/car/subaru/interface.py]: removed

**Comment by adeebshihadeh** [comment]
Synced the route. This is good to merge after it's added to the README.

**Comment by martinl** [comment]
replay tests passed, codegov report upload failed

**Comment by adeebshihadeh** [comment]
Thanks for the car port!

**Pull request merged**